src/app/components/progress-indicator/waterfall-progress-indicator/index.html

```diff
@@ -0,0 +1,325 @@
+<sky-tabbed-demo-page
+  pageTitle="Waterfall Progress Indicator"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The progress indicator module visually represents progress through sequential steps toward a goal. Waterfall progress indicators walk users through discrete steps on a page.
+    </p>
+  </sky-demo-page-summary>
+
+  <div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+      <sky-progress-indicator-demo>
+      </sky-progress-indicator-demo>
+    </sky-demo-page-example>
+  </div>
+
+  <sky-tabset id="content-tabs">
+    
+    <sky-tab
+      tabHeading="Design">
+    
+      <sky-demo-page-content sectionHeading="Usage">
+        <h3 class="sky-text-success">Use when</h3>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use waterfall progress indicators for long or complex setup processes that users typically complete only once. Waterfall indicators are ideal when events outside of the system force users to complete tasks in multiple sittings or when users must enter large amounts of data.</p>
+            <p>Use waterfall progress indicators when processes include at least three steps and the number and order of steps do not change.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-1.png" />
+              <div class="caption caption-success sky-text-success">Do use waterfall progress indicators for complex configuration processes that users may complete in stages.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Use waterfall progress indicators when the complexity of steps varies drastically. For example, one step may require a full-screen modal for complex interactions, while another step requires a small modal for a few simple inputs. Waterfall steps can open <a routerLink="/components/wizard">wizards</a> if necessary.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-separate">
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-2.png" />
```

Comment:
Why don’t the links in steps 1 and 2 and the button in step 3 line up? In the previous image and the anatomy example, the buttons line up with the links. However, the links and buttons don’t line up in the demo. It seems like the spacing should be consistent. Do we have a problem with inconsistent spacing in the way the component works? Or maybe a problem with the Design images? … More or less the same question about the second image of the second Use when item and the link in step 1 and the button in step 3 line up. And same question for the link and button in the first image of the first Don’t use when item. And in the first image of the third Don’t use when item, we have 4 links, and only 2 line up. That doesn’t seem right.

Reply:
I think you're asking about the horizontal positions of the actions inside the waterfall.

The component _centers_ those actions to the page (well, to its <div> I think, but _that's_ centered to the page as well). There's different horizontal space between the step indicators/labels and the step actions, but that's intentional.

Reply:
Sounds like this is probably working as intended. It looked like some links and action buttons were lined up on their lefthand starting point (i.e., the first letters of the links and the left edge of buttons) while others weren't. But if the items are centered, then I guess it's just that some labels look like they line up because their text is of a similar length.